.github/workflows/cli.yaml

```diff
@@ -0,0 +1,63 @@
+name: cli
```

Comment:
Moved the CLI tests to a separate action that can run parallel since it doesn't require image builds.